platform_versioned_sidebars/version-23.4.0-sidebars.json

```diff
@@ -7,24 +7,25 @@
       "collapsed": true,
       "items": [
         "getting-started/overview",
-        "getting-started/deployment-options",        
+        "getting-started/deployment-options",
+        "getting-started/start-collaborating"              
+      ]
+    },
+    {
+      "type": "category",
+      "label": "User guide",
+      "collapsed": true,
+      "items": [
         {
           "type": "category",
-          "label": "Start collaborating",
+          "label": "Workspaces",
```

Comment:
I vote we name this section Administration or something similar again, this would be more consistent with the categorization of the rest of the user guide IA, and also better encapsulates what the section contains. There is org and user information here, in addition to workspaces.